app/views/exceptions/show.html.erb

```diff
@@ -1,29 +1,43 @@
 <% if @bs4 %>
   <div class="d-flex full-height justify-content-center align-items-center text-center">
     <div>
-      <div class="warning-icon icon" alt="warning" aria-hidden="true">&nbsp;</div>
-      <%= sanitize_html(l10n(
-            "exceptions.show",
-            contact_center_phone_number: EnrollRegistry[:enroll_app].settings(:contact_center_short_number).item,
-            contact_center_tty_number: EnrollRegistry[:enroll_app].setting(:contact_center_tty_number).item.to_s,
-            contact_center_name: EnrollRegistry[:enroll_app].setting(:contact_center_name).item,
-            login_url: root_url
-          )) %>
+      <div class="d-flex justify-content-center">
+        <div class="warning-icon icon" alt="warning" aria-hidden="true">&nbsp;</div>
+      </div>
+      <div class="d-flex full-height justify-content-center align-items-center text-center">
+        <div>
+          <div class="d-flex justify-content-center">
+            <div class="warning-icon icon" alt="warning" aria-hidden="true">&nbsp;</div>
```

Comment:
warning-icon is duplicated here, confirm this is intended functionality

Reply:
negative, actually the whole outer div was duplicated. fixed